packages/esm-patient-list-app/src/patient-table/patient-table.scss

```diff
@@ -38,10 +37,15 @@ div.tableOverride > div > div {
 .paginationOverride{
     background-color: $ui-background;
     border-bottom: solid 1px $ui-03 !important;
+    overflow-x:hidden;
 }
 
 .paginationOverride > div {
     background-color: $ui-background;
     border-bottom: 0px!important;
     border-top: 0px !important;
 }
+
+.link {
+    text-decoration: none;
+}
```

Comment:
Maybe I'm missing it due to the way GitHub displays diffs, but is this class used anywhere? I cannot spot it right now. If it is, just ignore this comment please. 🙏🏻

Reply:
https://github.com/openmrs/openmrs-esm-patient-management/pull/53/files#r731306957